cli/sql_util.go

```diff
@@ -58,13 +71,20 @@ func runQuery(db *sql.DB, query string, parameters ...interface{}) (
 // found in the map are run through the corresponding callback.
 func runQueryWithFormat(db *sql.DB, format fmtMap, query string, parameters ...interface{}) (
 	[]string, [][]string, error) {
-	rows, err := db.Query(query, parameters...)
+	if insertPattern.MatchString(query) {
```

Comment:
I want to re-do the "pretty" querying. ideally, statements should display the number of rows affected.
Leaving this refactor for later though.